cores/arduino/pins_arduino_var.h

```diff
@@ -25,14 +25,16 @@
 // Include board variant
 #include "variant.h"
 
+// Avoid pins number misalignment
+_Static_assert(NUM_DIGITAL_PINS==PEND, "NUM_DIGITAL_PINS and PEND differ!");
+_Static_assert(NUM_ANALOG_INPUTS==(AEND-A0), "NUM_DIGITAL_PINS and (AEND-A0) differ!");
```

Comment:
Sorry made a wrong copy/paste
"~NUM_DIGITAL_PINS~NUM_ANALOG_INPUTS and (AEND-A0) differ!"

Reply:
No problem :)